droid-hal-version.inc

```diff
@@ -77,7 +77,6 @@ BuildRequires: droid-config-pulseaudio-settings
 # Haptics native
 %if 0%{?have_vibrator_native:1} && 0%{!?have_vibrator:1} && 0%{!?have_ffmemless:1}
 BuildRequires: ngfd-plugin-native-vibrator
-BuildRequires: qt5-feedback-haptics-native-vibrator
```

Comment:
I think we should replace qt5-feedback-haptics-native-vibrator with libngf-qt5-qtfeedback, not just delete old one.

Reply:
But one way to see the situation is that this droid-hal-version is supposed to depend on adaptation specific packages only and that libngf-qt5-qtfeedback is not one of those, but if we would have only those then we could remove also qtscenegraph-adaptation which used to be in adaptation builds but it not there anymore

Reply:
> I think we should replace qt5-feedback-haptics-native-vibrator with libngf-qt5-qtfeedback, not just delete old one.

Fixed that.